<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimButton</class>
 <widget class="QWidget" name="AnimButton">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>35</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>35</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="btStyle" native="true">
     <property name="styleSheet">
      <string notr="true">#btStyle:hover
{
	background: rgba(196, 243, 255, 150);
}</string>
     </property>
     <widget class="QWidget" name="lineBox" native="true">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>30</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">.QLabel
{
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <widget class="QLabel" name="line2">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>0</y>
         <width>3</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>3</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>3</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="line3">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>0</y>
         <width>3</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>3</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>3</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="line4">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>0</y>
         <width>3</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>3</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>3</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="line1">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>0</y>
         <width>3</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>3</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>3</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="text">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>8</y>
        <width>90</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="icon">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>8</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>lineBox</zorder>
     <zorder>icon</zorder>
     <zorder>text</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
